src/quo/animated.cljs

```diff
@@ -80,32 +110,39 @@
           :bouncyFactor              1
           :restSpeedThreshold        0.001
           :restDisplacementThreshold 0.001}})
-
+;
 (defn set-value
   [anim v]
   (ocall anim "setValue" v))
 
-(def Value (oget animated "Value"))
-
-(defn value
-  [x]
-  (new Value x))
-
-(def Clock (oget animated "Clock"))
-
-(defn clock
-  []
-  (new Clock))
-
-(def debug (oget animated "debug"))
-(def log (oget animated "log"))
-
+;;;; commented out to upgrade react-native-reanimated to v3 and react-native to 0.72
```

Comment:
We should try to respect the Lisp tradition for comments, it's in the Clojure Style Guide too.

- 4 and 3 `;;;;` are sections/headlines, they should be in 1 line only.
- 2 `;;` is for line comments
- 1 `;` is for margin comments

But as I shared in another comment, I think **animations in old code should be erased from existence** 💥 There's no point in maintaining them, only the functionality is important while we don't migrate them.